web/src/beta/features/Editor/Map/SketchLayerCreator/type.ts

```diff
@@ -10,7 +9,8 @@ export type SketchLayerDataType =
   | "Asset"
   | "Float"
   | "Int"
-  | "Boolean";
+  | "Boolean"
+  | "Camera";
```

Comment:
```suggestion
;
```
We don't need to support Camera at this time point i think.